js/project-page-functions.js

```diff
@@ -112,8 +112,14 @@ function onProjectPageLoaded() {
     }
   });
   var $body = $('body');
-  $body.scrollspy({target: '#right-sidebar-nav', offset: navHeight});
-  $body.scrollspy({target: '#left-sidebar-nav', offset: navHeight});
+  $body.scrollspy({'target': '#right-sidebar-nav', 'offset':navHeight});
+  // Offset in the above for some reason doesn't work, so we fix it this way with a little hack:
```

Comment:
I just didn't think that was necessary since we only use it for each revision. What are we scroll spying in the left-sidebar? I thought maybe we had that at one time not knowing what each sidebar would hold. From my bit of research of scrollspy, specifying a target container is only needed if there are multiple places that have the #<tag> hrefs. Let me know if I'm wrong in that regards. It has nothing to do with the "affix" thing we do with the left sidebar.